src/Microsoft.DotNet.CodeFormatting.Tests/Analyzers/ExplicitVariableTypeAnalyzerTests.cs

```diff
@@ -0,0 +1,237 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.CodeFormatting.Analyzers;
+
+using Xunit;
+
+namespace Microsoft.DotNet.CodeFormatting.Tests
+{
+    public sealed class ExplicitVariableTypeAnalyzerTests : AnalyzerFixerTestBase
+    {
+        public ExplicitVariableTypeAnalyzerTests()
+        {
+            DisableAllDiagnostics();
+            EnableDiagnostic(ExplicitVariableTypeAnalyzer.DiagnosticId);
+        }
+        [Fact]
+        public void TestSimpleTypeExplicitDeclaration()
+        {
+            var text = @"
+class C1
+{
+    void M()
+    {
+        int x = 0;
+        bool[] y = { true };  
+        int[][] z = new[] { new[] { 1, 2, 3 }, new[] { 4, 5, 6 } };
+    }
+}";
+            var expected = @"
```

Comment:
Since you expect no change, you can save typing by just setting expected = test. I had to read the whole "expected" string to convince myself there was no change.

Also I'd make those strings const. (I see I didn't do that in mine, either.)